dietpi/dietpi-software

```diff
@@ -11484,6 +11594,14 @@ _EOF_
 			rm -R "$FP_DIETPI_USERDATA_DIRECTORY"/nzbget
 			rm /etc/systemd/system/nzbget.service
 
+		elif (( $1 == 152 )); then
+
+			rm -R "$FP_DIETPI_USERDATA_DIRECTORY"/htpc-manager
+			
+			systemctl disable htpc-manager.service			
```

Comment:
Not required, as ```dietpi-services``` will disable this service during install (to take control), however, no harm.